src/components/pages/Login.tsx

```diff
@@ -1,7 +1,28 @@
-import { ActionButton, Input } from "..";
+import React, { useState } from 'react';
+import { ActionButton, Input } from "../..";
 import Circle from "../../assets/svgs/circle.svg";
+import axios from 'axios';
 
 function Login() {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const handleLogin = async () => {
+        
+            const url = `${base_url}/api/auth/signin`;
```

Comment:
@AyshaHakeem Could you please merge the current develop to your branch. After that please refer `src/services/api` directory and see how the other APIs are being integrated and change this according to that. Also, once you get the `success` response, please capture the `accessToken` in a global state/redux/context which ever is available as of now. We will have to send this `accessToken` as the authorization header for other APIs.